index.d.ts

```diff
@@ -0,0 +1,39 @@
+// copied from Redux so that we don't have to add it as a dependency
+interface Action<T = any> {
+  type: T
+}
+
+export type Enhancement = object;
+
+export type EnhancerId = string | symbol;
+
+type ActionEnhancerById = {
+  readonly id: EnhancerId;
+};
+
+type ActionEnhancerByType = {
+  actionTypes: string[];
+};
+
+export type ActionEnhancer<S, E extends Enhancement, V = any> = (
```

Comment:
Default `V` (enhancer value) to `any` since the most common use case is to not use this.

Reply:
should it be  unknown then?

Reply:
We could do `unknown` to force somebody who doesn't specify to further annotate the type before using it. We could also default it to a `boolean` since that seems to be the most common case.

Reply:
i like unknown. seems to provide the right amount of type safety. it just doesn't matter unless you care, in which  case you need to specify it. But if you want to use a value of 123 by default instead of true, that's fine it dosesn't really matter.

Reply:
Changed to `unknown`